guides/common/modules/proc_generating-a-report-for-installable-and-applicable-errata.adoc

```diff
@@ -0,0 +1,14 @@
+[id="Generating_a_Report_for_Installable_and_Applicable_Errata_{context}"]
+= Generating a Report for Installable and Applicable Errata
+
+Use the following procedure to view a list of installable and applicable errata to install.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Monitor* > *Report Templates*.
+. To the right of the report template that you want to use, click *Generate*.
```

Comment:
```suggestion
. For the *Host {endash} Applicable Errata* template, click *Generate*.
```

Still not happy with "For the ..."; maybe you can find a better wording here.

Reply:
Reworded it to `. Click *Generate* for the *Host {endash} Applicable Errata* template.`